File: Generar archivos/generarArchivo.rb
time = Time.now.to_f
file = File.open('10M.txt', 'w')
n = 10000*1000

for i in 1...n do
    s = "line"+(i.to_s)
    file.puts s
end
file.close
time = Time.now.to_f - time
puts time

File: Generar archivos/generarArchivo.cpp
#include<bits/stdc++.h>

using namespace std;

int main ()
{
    auto start = std::chrono::system_clock::now();
    int n = 1000*1000;
    freopen("1M.txt","w",stdout);
    for(int i = 0; i < n; i++){
        cout<<"line"<<i<<"\n";
    }
    auto end = chrono::system_clock::now();
    chrono::duration<double> elapsed_seconds = end-start;
    cout<<"Elapsed time: "<<elapsed_seconds.count()<<"\n";
    fclose(stdout);
    return 0;
}